docs/deployment_guide/partner_editable/pre_deployment.adoc

```diff
@@ -39,8 +42,143 @@ This Quick Start requires:
+
+=== Customize the {partner-product-short-name} configuration
+
+If needed, customize the {partner-product-short-name} configuration. The initial {partner-product-short-name} configurations are passed through the https://github.com/f5networks/f5-bigip-runtime-init[F5 BIG-IP Runtime Init^] configuration files. These files are the primary mechanisms to customize the initial {partner-product-short-name} configurations for your deployment. They also allow for large amounts of {partner-product-short-name} customizations. Customizing the {partner-product-short-name} configurations generally involves updating and rehosting these files and passing their location through the *bigIpRuntimeInitConfig* template parameters.
+
+By default, the Quick Start deploys a minimal 2NIC PAYG configuration into the default Quick Start network topology using the following configuration files in this repo's https://github.com/aws-quickstart/quickstart-f5-big-ip-virtual-edition-ha/tree/main/declarations[/declarations^] folder:
+
+** `runtime-init-conf-2nic-payg-instance01.yaml`
+** `runtime-init-conf-2nic-payg-instance02.yaml`
+
+A minimal baseline high availability (HA) failover cluster is deployed. No virtual services are provisioned.
+
+Optionally, you can deploy an example web application firewall (WAF)-protected web service to illustrate how services work. The example application deploys web application instances and a WAF-protected virtual service on the {partner-product-short-name} instances. To deploy the example application, set the *provisionExampleApp* parameter to *true* and reference the following configuration files via *bigIpRuntimeInitConfig* parameters:
+
+** `runtime-init-conf-2nic-payg-instance01-with-app.yaml`
+** `runtime-init-conf-2nic-payg-instance02-with-app.yaml`
+
+If your deployment requires additional customizations, update and rehost your configuration files at a custom URL.
+
+NOTE: *IMPORTANT* If hosting on GitHub, URLs that point to GitHub must use the raw file format (for example, `raw.githubusercontent.com`).
+
+The {partner-product-short-name} configurations must match the external environment in which they are deployed. Although the provided Runtime-Init configuration examples extract and templatize many common values, some values are still hardcoded and must be updated to match your specific environment or deployment first.
+
+==== Common customization example 1
+
+To deploy a Bring Your Own License (BYOL) instance, perform these steps:
+
+[arabic]
+. Edit or modify the Declarative Onboarding (DO) declaration in a
+corresponding example `byol` runtime-init configuration file with the new `regKey`
```

Comment:
Unclear run-on sentence:
_Edit or modify the Declarative Onboarding (DO) declaration in a corresponding example `byol` runtime-init configuration file with the new `regKey` value._